src/hypothesis/extra/datetime.py

```diff
@@ -276,6 +276,30 @@ def datetimes(allow_naive=None, timezones=None, min_year=None, max_year=None):
     )
 
 
+def dates(min_year=None, max_year=None):
+    """Return a strategy for generating dates."""
+    return DatetimeStrategy(
+        allow_naive=True, timezones=[],
+        min_year=min_year, max_year=max_year,
+    ).map(datetime_to_date)
+
+
+def datetime_to_date(dt):
+    return dt.date()
+
+
+def times():
```

Comment:
datetime.times does support timezones and it would be nice to expose that with the same set of arguments that are present in datetimes().